package.json

```diff
@@ -22,20 +22,20 @@
     "@types/jest": "^23.3.9",
     "@types/jira-client": "^6.4.0",
     "@types/lodash": "^4.14.121",
-    "@types/node": "^10.12.9",
+    "@types/node": "^14.6.4",
     "@types/node-fetch": "^2.1.3",
     "danger": "^7.0.9",
     "danger-plugin-spellcheck": "^1.2.3",
     "danger-plugin-yarn": "^1.3.0",
     "github-webhook-event-types": "^1.2.1",
     "husky": "^0.14.3",
-    "jest": "^24.3.0",
+    "jest": "^26.4.2",
     "jira-client": "^6.4.1",
     "lint-staged": "^7.2.2",
-    "prettier": "^1.15.2",
-    "ts-jest": "^24.0.0",
+    "prettier": "^2.1.1",
+    "ts-jest": "^26.3.0",
     "ts-node": "^7.0.1",
-    "typescript": "^3.1.6"
+    "typescript": "3.4.1"
```

Comment:
**question [non-blocking]:** typescript version pin

This is the pin you mentioned? Couldn't one clobber this with `yarn upgrade --latest`? Is there another, more resilient way to specify which version of a package one wants? 🤔 

*format via [Conventional Comments](https://conventionalcomments.org/)*

Reply:
This is the pin, I'm not sure about the semantics of `yarn upgrade` – I'm basing this off Renovate's PRs to pin dependencies in Eigen, which follow this format. (Previously, in c2cdda6, this was at 4.0.2.)